src/App.js

```diff
@@ -26,40 +28,44 @@ const App = () => {
   };
 
   return (
-    <form onSubmit={checkUserAnswers}>
-      {data.map(question => {
-        return (
-          <fieldset key={question.id}>
-            <legend>{question.text}</legend>
-
-            {question.answers.map(answer => {
-              return (
-                <div key={answer.id}>
-                  <input
-                    type="radio"
-                    id={`${question.id}.${answer.id}`}
-                    name={question.id}
-                    value={answer.id}
-                    onClick={handleAnswerSelect}
-                  />
-                  <label htmlFor={`${question.id}.${answer.id}`}>
-                    {answer.text}
-                  </label>
-                </div>
-              );
-            })}
-          </fieldset>
-        );
-      })}
-      {Object.keys(userAnswers).length === data.length && (
-        <button>Submit</button>
-      )}
-      {shouldDisplayScore && (
-        <h2>
-          You have got {score} out of {data.length}
-        </h2>
-      )}
-    </form>
+    <Container>
+      <Row>
+        <Col> <h1 style={{ textAlign: "center" }} >CYF Quiz</h1>  </Col>
+      </Row>
+      <Form onSubmit={checkUserAnswers}>
+        {data.map(question => {
+          return (
+            <Jumbotron>
+              <Form.Group key={question.id}>
+                <legend>{question.text}</legend>
```

Comment:
We can probably change `legend` to `h2` because we removed `fieldset`, and `legend` is normally only used with `fieldset`. 

Also, `legend`/`h2` should go above `Form.Group` since a form group should contain inputs, labels and normal text only.